tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -36,7 +36,7 @@
 
 T = TypeVar('T')
 
-REJECTED_METRIC = "vectorized_kafka_rpc_connections_rejected_total"
+REJECTED_METRIC = "vectorized_kafka_rpc_connections_rejected"
```

Comment:
So the function we call with this was just silently returning 0?

Reply:
Can you fix this in tests/rptest/tests/connection_limits_test.py too?

Reply:
> So the function we call with this was just silently returning 0?

Yeah metrics_sum will return 0 if not found.